src/UserConfig.elm

```diff
@@ -1651,6 +1651,411 @@ Podemos cerrar su cuenta de inmediato si pensamos que ha:
+    , thisGroupWasCreatedOn = "กลุ่มนี้ถูกสร้างขึ้นเมื่อ "
+    , timeDiffToString = diffToStringEnglish
+    , timeValueMissing = "ไม่มีค่าเวลา"
+    , title = "กิจกรรม"
+    , tos = "ข้อกำหนดในการให้บริการ"
+    , tosMarkdown = \privacyRoute codeOfConductRoute -> """
+
+#### รุ่น 1.0 - มิถุนายน 2021
+
+### 🤔 Meetdown คืออะไร
+
+เงื่อนไขกฎหมายเหล่านี้คือสิ่งที่คุณและ meetdown.app (“เรา”, “ของเรา”, “Meetdown”, ซอฟต์แวร์) ตกลงกัน และคุณตกลงตามเงื่อนไขเหล่านี้โดยการใช้บริการ Meetdown
+
+คุณควรอ่านเอกสารนี้พร้อมกับ [Data Privacy Notice](""" ++ privacyRoute ++ """) ของเรา
+
+### 💬 วิธีติดต่อเรา
+
+โปรดแชทกับเราโดยส่งอีเมลถึงเราที่ [""" ++ Env.contactEmailAddress ++ """](mailto:""" ++ Env.contactEmailAddress ++ """)
+
+เราจะติดต่อคุณในภาษาอังกฤษ 🇬🇧 และอีโมจิ 😃
+
+### 🤝🏽 การรับประกันและความคาดหวัง
+
+Meetdown ไม่มีการรับประกัน
+
+[โค้ดที่มาสำหรับ Meetdown](https://github.com/MartinSStewart/meetdown) เป็นโอเพนซอร์สดังนั้นผู้ใช้เทคนิคอาจทำการประเมินความเสี่ยงเอง
+
+ซอฟต์แวร์ให้บริการ "ตามที่เป็น" โดยไม่มีการรับประกันอย่างใด ๆ ไม่ว่าจะชัดแจ้งหรือโดยการอนุมัติ รวมถึงแต่ไม่จำกัดเพียงการรับประกันของการขาย การความเหมาะสมสำหรับวัตถุประสงค์ที่กำหนด และการไม่ละเมิดสิทธิ
+
+ในกรณีใด ๆ ก็ตาม ผู้เขียนหรือผู้ถือลิขสิทธิ์จะไม่รับผิดชอบต่อคำร้องขอ ความเสียหายหรือความรับผิดชอบอื่น ๆ ไม่ว่าจะในการดำเนินคดีของสัญญา ความผิดหรืออื่น ๆ ที่เกิดจาก โดยมาจากหรือในการเชื่อมโยงกับซอฟต์แวร์หรือการใช้หรือการเจรจาอื่น ๆ ในซอฟต์แวร์
+
+เราคาดหวังว่าผู้ใช้ทุกคนจะปฏิบัติตาม [Code of conduct](""" ++ codeOfConductRoute ++ """)
+
+
+### 💵 ค่าใช้จ่าย
+
+Meetdown เป็นผลิตภัณฑ์ที่ให้บริการฟรี
+
+
+### 😔 วิธีการยื่นเรื่องร้องเรียน
+
+ถ้าคุณมีเรื่องร้องเรียน กรุณาติดต่อเราและเราจะทำทุกอย่างเพื่อแก้ไขปัญหา
+
+โปรดดู "วิธีติดต่อเรา" ด้านบน
+
+
+### 📝 การทำการเปลี่ยนแปลงข้อตกลงนี้
+
+ข้อตกลงนี้จะสามารถใช้ได้ตลอดเวลาที่ meetdown.app
+
+หากเราทำการเปลี่ยนแปลง คุณจะได้รับการแจ้งเตือนทันทีหลังจากที่เราทำการเปลี่ยนแปลง
+
+หากคุณไม่เห็นด้วยกับการเปลี่ยนแปลงเหล่านี้ คุณสามารถปิดบัญชีของคุณได้โดยการกด "Delete Account" ที่หน้าโปรไฟล์ของคุณ
+
+เราจะทำลายข้อมูลในบัญชีของคุณ ยกเว้นกรณีที่เราจำเป็นต้องเก็บไว้ตามที่กำหนดไว้ใน [Privacy policy](""" ++ privacyRoute ++ """)
+
+
+### 😭 การปิดบัญชีของคุณ
+
+เพื่อปิดบัญชีของคุณ คุณสามารถกดที่ปุ่ม "Delete Account" ที่หน้าโปรไฟล์ของคุณ
+
+เราสามารถปิดบัญชีของคุณโดยให้คุณรับทสายเป็นอย่างน้อยหนึ่งสัปดาห์
+
+เราอาจปิดบัญชีของคุณทันทีถ้าเราเชื่อว่าคุณ:
+
+- ฝ่าฝืนข้อตกลงในสัญญานี้
+- ทำให้เราอยู่ในสถานการณ์ที่เราอาจฝ่าฝืนกฎหมาย
+- ฝ่าฝืนกฎหมายหรือพยายามฝ่าฝืนกฎหมาย
+- ให้ข้อมูลที่ไม่จริงกับเราที่ใด ๆ เวลา
+- มีพฤติกรรมทารุณโต้เถียงกับบุคคลใด ๆ ที่ Meetdown หรือสมาชิกในชุมชนของเรา
+
+"""
+    , twoPeopleOnAVideoConference = "สองคนที่กำลังประชุมผ่านวิดีโอ"
+    , uncancelEvent = "ยกเลิกการยกเลิกกิจกรรม"
+    , uploadImage = "อัปโหลดรูปภาพ"
+    , userNotFound = "ไม่พบผู้ใช้"
+    , valueMustBeGreaterThan0 = "ค่าต้องมากกว่า 0."
+    , weDontSellYourDataWeDontShowAdsAndItsFree = "เราไม่ขายข้อมูลของคุณ เราไม่แสดงโฆษณา และมันฟรี."
```

Comment:
the word "it" translated as "มัน" reads weirdly…

```suggestion
    , weDontSellYourDataWeDontShowAdsAndItsFree = "เราไม่ขายข้อมูลของคุณ เราไม่แสดงโฆษณา และบริการนี้ฟรีี"
```

reworded to "this service is free"